internal/util/logging/console_handler.go

```diff
@@ -102,10 +96,18 @@ func (ch *consoleHandler) Handle(ctx context.Context, r slog.Record) error {
 		buf.WriteRune('\t')
 
 		if ch.testAttrs != nil {
-			ch.testAttrs["level"] = l
+			ch.testAttrs[slog.LevelKey] = l
 		}
 	}
 
+	m, name := attrs(r, ch.ga)
+
+	// TODO https://github.com/FerretDB/FerretDB/issues/4431
+	if name != "" {
+		buf.WriteString(name)
+		buf.WriteRune('\t')
+	}
```

Comment:
Let's not try to change name handling in this PR – it is already quite complicated (and in progress for a long time), and that's a separate issue anyway